.github/workflows/image.yaml

```diff
@@ -43,3 +43,51 @@ jobs:
     secrets:
       registry_username: ${{ github.actor }}
       registry_password: ${{ secrets.CR_PAT }}
+
+  generate-init-kyverno-provenance:
+    needs: push-init-kyverno
+    permissions:
+      id-token: write # To sign the provenance.
+      packages: write # To upload assets to release.
+      actions: read #To read the workflow path.
+    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@main
```

Comment:
Please use tagged releases for all calls to the generator.

Reply:
I tested v1.2.0 and it may be broken, so I think `main` is OK for now because it hasn't officially been released.

Reply:
Do you mean using some specific release version of the container generator instead of main?

Reply:
Yes, however it hasn't been officially released yet. I tried with v1.2.0 and saw errors on my end and the only version which worked came from `main`.